bluemix/configuration/core_config/iam_token.go

```diff
@@ -82,3 +84,16 @@ func decodeAccessToken(token string) (tokenJSON []byte, err error) {
 	encodedTokenJSON := encodedParts[1]
 	return base64.RawURLEncoding.DecodeString(encodedTokenJSON)
 }
+
+// HasExpired returns True if the token expiry has occured
+// before today + delta time or a token is invalid
+func (t IAMTokenInfo) HasExpired() bool {
+	// We can assume that a token without an ID is invalid and expired
+	if t.ID == "" {
+		return true
```

Comment:
If we change the check I mentioned above, I think this will need to return false instead.